aggregate_root/lib/aggregate_root.rb

```diff
@@ -4,6 +4,22 @@
 require 'aggregate_root/default_apply_strategy'
 
 module AggregateRoot
+  module ClassMethods
+    def on(*event_klasses, apply_strategy:  DefaultApplyStrategy.new, &block)
```

Comment:
The fact that it gets `apply_strategy` as an argument is a big smell for me. After all https://github.com/RailsEventStore/rails_event_store/blob/0f7ee713a08d9b834c28cbfe25c00d995e3d8b64/aggregate_root/lib/aggregate_root.rb#L47 the strategy can be overwritten on an instance level, not class level so there is a mismatch here. These two features don't place nicely together.

Reply:
to be honest, I found this issue as well, for me stragegy can be class configuration, not instance method

Reply:
but I can't refactor it, without breaking backwad compatibility

Reply:
One month has passed and we haven't figure out what to do about it.